.circleci/scripts/bundle.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+export CIRCLE_BRANCH=""
+export CONTENTFUL_ACCESS_SPACE_ID=""
+export CONTENTFUL_ACCESS_TOKEN=""
+export ETHERSCAN_API_KEY=""
+export FIREBASE_API_KEY=""
+export FIREBASE_APP_ID=""
+export FIREBASE_AUTH_DOMAIN=""
+export FIREBASE_MEASUREMENT_ID=""
+export FIREBASE_MESSAGING_SENDER_ID=""
+export FIREBASE_PROJECT_ID=""
+export FIREBASE_STORAGE_BUCKET=""
+export INFURA_BETA_PROJECT_ID=""
+export INFURA_FLASK_PROJECT_ID=""
+export INFURA_MMI_PROJECT_ID=""
+export INFURA_PROD_PROJECT_ID=""
+export INFURA_PROJECT_ID=""
+export MMI_CONFIGURATION_SERVICE_URL=""
+export PUBNUB_PUB_KEY=""
+export SEGMENT_BETA_WRITE_KEY=""
+export SEGMENT_FLASK_WRITE_KEY=""
+export SEGMENT_MMI_WRITE_KEY=""
+export SEGMENT_PROD_LEGACY_WRITE_KEY=""
+export SEGMENT_PROD_WRITE_KEY=""
+export SEGMENT_WRITE_KEY=""
+export ANALYTICS_DATA_DELETION_SOURCE_ID=""
+export ANALYTICS_DATA_DELETION_ENDPOINT=""
+export SENTRY_AUTH_TOKEN=""
+export SENTRY_DSN=""
+export SENTRY_MMI_DSN=""
+export SENTRY_DSN_DEV=""
+export TZ=""
+export VAPID_KEY=""
+export ENABLE_MV3=""
```

Comment:
I think ENABLE_MV3 is one that we don't keep in circleci. Instead, because this is for firefox submission, it will alwats be true